frontend/src/app/app-routing.module.ts

```diff
@@ -19,12 +22,35 @@ const routes: Routes = [
   },
 
   // Experiments
-  { path: 'experiments', component: ExperimentsListComponent, canActivate: [AuthGuard] },
-  { path: 'experiments/create', component: ExperimentCreateComponent, canActivate: [AuthGuard] },
+  { 
+    path: 'experiments', 
+    component: ExperimentListComponent, 
+    canActivate: [AuthGuard] 
+  },
+  { 
+    path: 'experiments/create', 
+    component: ExperimentCreateComponent, 
+    canActivate: [AuthGuard] 
+  }, 
+  { 
+    path: 'experiments/:id', 
+    component: ExperimentRetrieveComponent, 
+    canActivate: [AuthGuard] 
+  },
+  { 
+    path: 'experiments/update/:id', 
+    component: ExperimentUpdateComponent, 
+    canActivate: [AuthGuard] 
+  },  
+  { 
+    path: 'experiments/delete/:id', 
+    component: ExperimentDeleteComponent, 
+    canActivate: [AuthGuard] 
+  },
   // All other routes
   { 
     path: '**', 
-    redirectTo: '' 
+    redirectTo: 'experiments' 
```

Comment:
Why is the redirect to /experiments here?

Reply:
I think that it's a good idea because we don't have a "Home" view now and the Experiments.List could be that for now.